### PR TITLE
Import build definitions

### PR DESCRIPTION
**Pull request by exocom**
## All Submissions:
-------------------------------------
* [YES] Have you added an explanation of what your changes do and why you'd like us to include them?
* [YES] I have updated the documentation accordingly.
* [YES] I have added tests to cover my changes.
* [YES] All new and existing tests passed.
* [YES] My code follows the code style of this project.
* [YES] I ran lint checks locally prior to submission.
* [YES] Have you checked to ensure there aren't other open PRs for the same update/change?

## What is the current behavior?
-------------------------------------

Currently build definitions can NOT be imported. This adds the ability to import them.

Issue Number: N/A

## What is the new behavior?
-------------------------------------

- You can import build definitions using terraform import
- Path must start with \
- Path can not be an empty string

## Does this introduce a breaking change?
-------------------------------------
- [NO]

## Any relevant logs, error output, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

## Other information
-------------------------------------

### CODE REVIEW

File: azuredevops/utils/tfhelper/tfhelper.go [hunk 1]
```diff
@@ -95,3 +95,14 @@ func ParseImportedID(id string) (string, int, error) {
...
 }
+
+// ParseImportedUUID parse the imported uuid from the terraform import
+func ParseImportedUUID(id string) (string, string, error) {
+	parts := strings.SplitN(id, "/", 2)
+	if len(parts) != 2 || parts[0] == "" || parts[1] == "" {
+		return "", "", fmt.Errorf("unexpected format of ID (%s), expected projectid/resourceId", id)
+	}
+	project := parts[0]
+	uuid := parts[1]
```
Comment (tmeckel) [inline azuredevops/utils/tfhelper/tfhelper.go]: Perhaps the function should return a parsed UUID and if the string from parts[1] isn't a valid UUID should return an error respectively.

Comment (exocom) [reply azuredevops/utils/tfhelper/tfhelper.go]: All set, it now returns byte[] rather than string.

**Comment by nmiodice** [comment]
@exocom, thank you for the contribution. Looks good to me. There are a handful of merge conflicts. Can you have a look and re-push so that I can do a clean merge?

**Comment by exocom** [comment]
@nmiodice Yes, and thank you! The build is running now.

**Comment by nmiodice** [comment]
@exocom I see the following error in the acceptance tests run:
```
2020-01-21T18:06:59.4626329Z --- FAIL: TestAccAzureDevOpsServiceEndpointGitHub_CreateAndUpdate (25.45s)
2020-01-21T18:06:59.4626448Z     testing.go:569: Step 2 error: Error parsing the service endpoint ID from the Terraform resource data: invalid UUID length: 16
```

**Comment by exocom** [comment]
@nmiodice Thanks again. I've updated the code . Going back to using strings over uuid/byte array. There was something I'm missing when converting the byte array back to a string. Can always add this in a much smaller pr.

**Pull request merged**